core/che-core-db/src/main/java/org/eclipse/che/core/db/jpa/DuplicateKeyException.java

```diff
@@ -19,6 +19,7 @@
  * @author Yevhenii Voevodin
  * @see DBErrorCode#DUPLICATE_KEY
  */
+@SuppressWarnings("serial")
```

Comment:
The only warning that I found during maven build in `/core/che-core-db` module is 
```
[INFO] Compiling 27 source files to /home/serg/projects/che/core/che-core-db/target/classes                                                                                                                       
/home/serg/projects/che/core/che-core-db/src/main/java/org/eclipse/che/core/db/jpa/guice/GuiceJpaInitializer.java:33: warning: [MissingOverride] init implements method in JpaInitializer; expected @Override     
  public void init() {                                                                                                                                                                                            
              ^                                                                                                                                                                                                   
    (see http://errorprone.info/bugpattern/MissingOverride)
  Did you mean '@Override public void init() {'?
Note: /home/serg/projects/che/core/che-core-db/src/main/java/org/eclipse/che/core/db/jpa/eclipselink/GuiceEntityListenerInjectionManager.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 warning
```
Can you elaborate which warnings you suppress? What the cause of them? Maybe there is an ability to fix cause instead of suppressing warning?

Reply:
Warnings are shown on Eclipse/JDT (not Che).
I can suppress that warnings by settings. But I guess many IDE (except Che) will show this warning by default.

The reason why `@SuppressWarnings("serial")` should be added, it will be some helps to clarify whether "the author of the class decided not to care about serialize" or "he/she just forgot to add `serialVersionUID`."